CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## [0.0.36] - 2024-07-23
+
+### Added
+
+- [#342](https://github.com/Azure/draft/pull/342) Added k8s/deploy Inputs for Private Cluster Support
+- [#337](https://github.com/Azure/draft/pull/337) azure pipelines generation support
+- [#334](https://github.com/Azure/draft/pull/334) template support for private clusters
+- [#324](https://github.com/Azure/draft/pull/324) workflow template enhancements
+- [#321](https://github.com/Azure/draft/pull/321) add helm rendering function
+- [#315](https://github.com/Azure/draft/pull/315) adding logic for generating default app name
+
+### Fixed
+
+- Workflows are now generated fully from a draftConfig
```

Comment:
Should we add the link for the corresponding PR here? I think it's this one [#320](https://github.com/Azure/draft/pull/320)

Reply:
yep added